packages/databricks-sdk-js/src/config/oauth/Client.ts

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import {Token} from "../Token";
+import {Issuer} from "./Issuer";
+import {Headers} from "../Config";
+import fetch from "node-fetch-commonjs";
+import {RequestInit} from "node-fetch-commonjs";
+import {getBasicAuthHeader} from "../BasicCredentials";
+
+export interface ClientOptions {
+    clientId: string;
+    clientSecret: string;
+    useParams?: boolean;
+    useHeader?: boolean;
+    headers?: Headers;
+}
+
+export class Client {
+    constructor(private issuer: Issuer, private options: ClientOptions) {
+        options.useParams = options.useParams ?? false;
+        options.useHeader = options.useHeader ?? false;
+        options.headers = options.headers ?? {};
+    }
+
+    async grant(scope: string): Promise<Token> {
```

Comment:
can you call it `exchangeToken` to be more uniform across all SDKs?

Reply:
yes